packages/actor-query-operation-path-alt/lib/ActorQueryOperationPathAlt.ts

```diff
@@ -0,0 +1,38 @@
+import {ActorAbstractPath} from "@comunica/actor-abstract-path";
+import {
+  ActorQueryOperation,
+  IActorQueryOperationOutputBindings, IActorQueryOperationTypedMediatedArgs,
+} from "@comunica/bus-query-operation";
+import {ActionContext} from "@comunica/core";
+import {RoundRobinUnionIterator} from "asynciterator-union";
+import {Algebra} from "sparqlalgebrajs";
+
+/**
+ * A comunica Path Alt Query Operation Actor.
+ */
+export class ActorQueryOperationPathAlt extends ActorAbstractPath {
+
+  constructor(args: IActorQueryOperationTypedMediatedArgs) {
+    super(args, Algebra.types.ALT);
+  }
+
+  public async runOperation(path: Algebra.Path, context: ActionContext)
+    : Promise<IActorQueryOperationOutputBindings> {
+    const predicate = <Algebra.Alt> path.predicate;
+
+    const left = ActorQueryOperation.getSafeBindings(await this.mediatorQueryOperation.mediate({
```

Comment:
We can optimize this by moving the `await`'s to a higher level, so that left and right can run in parallel.